setup.cfg

```diff
@@ -1,7 +1,10 @@
 [flake8]
 max-line-length = 120
-max-complexity = 17
 exclude =
   .git,
-  .tox,
-  __pycache__
+  __pycache__,
+  build/,
+  dist/,
+  docs
+per-file-ignores =
+  nwbinspector/__init__.py:F401,F403
```

Comment:
The CI now uses `black`, which is configured via the `pyproject.toml` (more modern and more broadly supported PEP518 config file) file instead of `setup.cfg` (see this delightfully long conversation about why `flake8` won't support it: https://github.com/PyCQA/flake8/issues/234), but I've updated the `setup.cfg` anyway if someone wants to generate a flake8 report separately.